main.go

```diff
@@ -24,7 +24,8 @@ import (
 func main() {
 	// TODO: detect test runner and use appropriate runner
 	testRunner := runner.Rspec{}
-
+	// Get the arguments without the program name
+	argsWithoutProg := os.Args[1:]
```

Comment:
This gets all args, which includes flags (e.g. if someone uses the `--files` flag then this will contain `"--files"` somewhere). `flag.Args()` returns the program args not including flags, which you will need to call after `flag.Parse()`.

Reply:
Hey @DrJosh9000 thanks for calling this out
We are going to stop supporting the `--files` flags as we are not using it anymore. 
I'm wondering if we remove the support of `--files` flag, do I still need to use `flag.Args()` for the program args?

Reply:
I think you shouldn't mix and match, and you should either:

* use `flag.Args()` in this PR, and replace it with `os.Args` when you get rid of `flag` in a later PR

or

* get rid of `flag` and change to `os.Args` in this PR.

Reply:
Thanks Josh, I'll get rid of flag and change to os.Args in this PR. I'll also update the PR description